scripts/prebuild.mjs

```diff
@@ -18,6 +18,34 @@ const dirFoldersOnly = async (dir, files) => {
   }
 };
 
+const matchNameNScripts = (dir, files) => {
+  for (let file of files) {
+    fs.stat(`${dir}${file}/package.json`, (err) => {
+      if (err) {
+        console.error(`❌ package.json file not found in ${dir}${file}`);
+        return;
+      }
+      const json = JSON.parse(
+        fs.readFileSync(`${dir}${file}/package.json`).toString()
+      );
+
+      if (json?.name !== `polkadotcloud-${file}`) {
+        console.error(
+          `❌ ${json?.name} package name doesn't meet the naming requirement`
+        );
+      }
+
+      if (
+        !Object.keys(json?.scripts).includes("build:mock" && "build" && "clear")
```

Comment:
I don't think this `includes` is being used correctly, the inner definition will result to `true`? use `indexOf` instead.